tests/test_dojo.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from classes.dojo import Dojo
+
+
+class DojoClassTest(unittest.TestCase):
+    def test_create_room_successfully(self):
+        my_class_instance = Dojo()
+        initial_room_count = len(my_class_instance.all_rooms)
+        blue_office = my_class_instance.create_room("office", "Blue")
+        new_room_count = len(my_class_instance.all_rooms)
+        self.assertEqual(new_room_count - initial_room_count, 1)
+
+    def test_create_room_without_duplicates(self):
+        my_class_instance = Dojo()
+        blue_office = my_class_instance.create_room("office", "Blue")
```

Comment:
You are creating variable `blue_office` but not using it. Consider having a room created in the `setUp` class, as it is repetetive